src/v2/Apps/Conversation/Components/__tests__/ConversationMessages.jest.tsx

```diff
@@ -11,6 +11,7 @@ const { renderWithRelay } = setupTestWrapperTL({
       <ConversationMessagesFragmentContainer
         messages={me.conversation.messagesConnection}
         events={me.conversation.orderConnection}
+        setShowDetails={() => {}}
```

Comment:
Would it make sense to set this to not only test if "See Details" is there, but what is triggering?

Reply:
Good call. I think it makes sense to test it 👍